guides/user/managed-delivery/index.md

```diff
@@ -2,7 +2,8 @@
 layout: single
 title:  "Quick Reference"
```

Comment:
```suggestion
title:  "Managed-Delivery Guide"
```

Reply:
I like the title "quick reference"

Reply:
Fair. But since it's the overview page for the whole guide, do you wanna put "Managed Delivery" in the title too?

Reply:
It's already under the managed delivery heading, so I don't think so. However, if it's confusing in the future, I'm happy to change it.